de-DE/Codex_der_Charaktere/06_Zauberwirken/Beschreibungen_der_Zauber/B/Brandmarkendes_Niederstrecken.md

```diff
@@ -4,42 +4,32 @@
 <!-- spell-checker:words branding smite -->
 [_metadata_:spell_name]:- "Brandmarkendes Niederstrecken"
 [_metadata_:spell_name_english]:- "Branding Smite"
-[_metadata_:spell_school]:- "???"
-[_metadata_:spell_level]:- "???"
-[_metadata_:casting_time_amount]:- "???"
-<!-- "action", "bonus action", "reaction", "minute", "minutes", "hour", "hours" -->
-[_metadata_:casting_time_unit]:- "???"
-[_metadata_:casting_time_reaction_trigger]:- "???"
-<!-- "true" oder "false" -->
+[_metadata_:spell_school]:- "Hervorrufungszauber"
+[_metadata_:spell_level]:- "2"
+[_metadata_:casting_time_amount]:- "1"
+[_metadata_:casting_time_unit]:- "Bonusaktion"
 [_metadata_:ritual]:- "false"
-[_metadata_:range]:- "???"
-[_metadata_:target]:- "???"
-<!-- "true" oder "false" -->
-[_metadata_:components_verbal]:- "???"
-<!-- "true" oder "false" -->
-[_metadata_:components_somatic]:- "???"
-<!-- "true" oder "false" -->
-[_metadata_:components_material]:- "???"
-[_metadata_:components_material_description]:- "???"
-[_metadata_:components_material_cost]:- "???"
-<!-- "true" oder "false" -->
-[_metadata_:concentration]:- "???"
-[_metadata_:duration]:- "???"
-[_metadata_:saving_throw]:- "???"
-<!-- "halves_damage", "avoids_effect", "ends_effect", "special"; falls mehrere wahr sind, kommagetrennt einfügen -->
-[_metadata_:saving_throw_success]:- "???"
-[_metadata_:damage_formula]:- "???d???"
-[_metadata_:damage_type]:- "???"
+[_metadata_:range]:- "Selbst"
+[_metadata_:target]:- "Selbst"
+[_metadata_:components_verbal]:- "true"
+[_metadata_:components_somatic]:- "false"
+[_metadata_:components_material]:- "false"
+[_metadata_:concentration]:- "true"
+[_metadata_:duration]:- "Konzentration, bis zu 1 Minute"
 [_metadata_:compared_to_wotc_srd_5.1]:- "mechanics_same_wording_same"
-<!-- "mechanics_same_wording_different", "mechanics_different_wording_different" oder "added" -->
-[_metadata_:compared_to_a5e_srd]:- "???"
+[_metadata_:compared_to_a5e_srd]:- "mechanics_same_wording_same"
 <!-- markdownlint-disable-next-line no-emphasis-as-heading -->
-<mark>_X-zauber n. Grades (Ritual?)_ oder _Zaubertrick der X_</mark>
+_Hervorrufungszauber 2. Grades_
 
-**Zeitaufwand:** ??? \
-**Reichweite:** ??? \
-**Komponenten:** ??? \
-**Wirkungsdauer:** ???
+**Zeitaufwand:** 1 Bonusaktion \
+**Reichweite:** Selbst \
+**Komponenten:** V \
+**Wirkungsdauer:** Konzentration, bis zu 1 Minute
 
-> **Warning**
-> Dieser Zauber wird noch übersetzt.
+Das nächste Mal, wenn du eine Kreatur mit einem Waffenangriff triffst, bevor der Zauber endet, umgibt die Waffe ein astrales Leuchten.
+
+Der Angriff fügt dem Ziel zusätzlich 2W6 gleißenden Schaden zu.
+
+Es wird sichtbar, falls es unsichtbar war, wirft dämmriges Licht in einem Umkreis von 1,5 Metern und kann nicht unsichtbar werden, bis der Zauber endet.
```

Comment:
We do not need line breaks in the converted file. Also, please put the damage formula in backticks.
```suggestion
Das nächste Mal, wenn du eine Kreatur mit einem Waffenangriff triffst, bevor der Zauber endet, umgibt die Waffe ein astrales Leuchten.
Der Angriff fügt dem Ziel zusätzlich `2W6` gleißenden Schaden zu.
Es wird sichtbar, falls es unsichtbar war, wirft dämmriges Licht in einem Umkreis von 1,5 Metern und kann nicht unsichtbar werden, bis der Zauber endet.
```
Also, in spells that mention conditions in the English version I link to those conditions. We don't currently have the conditions transferred, so we'll have to add that link later.